rc/extra/tmux-repl.kak

```diff
@@ -50,9 +50,15 @@ define-command tmux-repl-window -params 0..1 -command-completion -docstring "Cre
     tmux-repl-impl 'new-window' %arg{@}
 }
 
-define-command -hidden tmux-send-text -docstring "Send the selected text to the repl pane" %{
+define-command -hidden tmux-send-text -params 0..1 -docstring "Send the selected text to the repl pane if no parameter.
+  Otherwise send parameter with new line" %{
     nop %sh{
-        tmux set-buffer -b kak_selection "${kak_selection}"
+        if [ $# -eq 0 ]; then 
+            tmux set-buffer -b kak_selection "${kak_selection}"
+        else
+            tmux set-buffer -b kak_selection "$1
```

Comment:
There's a trailing newline before the ending quote, is that on purpose?

Reply:
Yes. For example, you can select a word and send with new line, which will print its value by most interpreters.
Do you know a better way to add new line?

Reply:
We should not assume that users only want to send whole commands to the REPL prompt, so I think the newline character should be removed. Users could declare a command that sends text followed by a newline, if they want that behavior.

Also, `$'\n'`.

Reply:
Agree. Consider it done.

Reply:
`$'\n'` is not posix :(

Reply:
In that case:

```bash
readonly NEWLINE=$(echo)
```